src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -3600,7 +3624,7 @@ End Class
 
         <Fact, Trait(Traits.Feature, Traits.Features.Formatting)>
         <WorkItem(529899, "DevDiv_Projects/Roslyn")>
-        Public Async Function IndentContinuedLineOfSingleLineLambdaToFunctionKeyword() As Task
+        Public Async Function IndentContinuedLineOfSingleLineLambdaToFunctionKeyword___() As Task
```

Comment:
why did you add ___ to all the tests?

Reply:
So I could identify all the tests that are effected by my changes, as I said it was temporary. I will put the old tests back (which was always my intent) and put my changes next to them with ___.

Reply:
I have restored all the only test, there are 3 possibilities, New  ___ tests passes and old test fails this is expected, the new test is what I think the code should look like.  Both tests fail, here the old test is expected to fail but my changes did not produce desired result as shown in new test. Old test fails and there is no new test, my changes broke the old code.